src/sentry/search/events/datasets/discover.py

```diff
@@ -1000,6 +1000,50 @@ def function_converter(self) -> Mapping[str, SnQLFunction]:
                     snql_aggregate=self._resolve_count_scores_function,
                     default_result_type="integer",
                 ),
+                SnQLFunction(
+                    "example",
+                    snql_aggregate=lambda args, alias: Function(
+                        "arrayElement",
+                        [
+                            Function(
+                                "groupArraySample(1, 1)",  # TODO: paginate via the seed
+                                [
+                                    Function(
+                                        "tuple",
+                                        [Column("timestamp"), Column("span_id")],
+                                    ),
+                                ],
+                            ),
+                            1,
+                        ],
+                        alias,
+                    ),
+                    private=True,
```

Comment:
What does private mean? It's not exposed to the outside meaning endpoints like `events-stats` or it's just not shown in the available ops?

Reply:
Private means this function is for internal use only, and users of the API must explicitly add this function to the `functions_acl` in the configs to be able to use this function.

This means other endpoints such as events-stats will not have access to this function.

Reply:
Got it, thanks.